deploy/cluster_role.yaml

```diff
@@ -48,3 +48,42 @@ rules:
       - update
       - patch
       - delete
+  - apiGroups:
+      - rbac.authorization.k8s.io
+    resources:
+      - clusterrolebindings
+    verbs:
+      - list
+      - watch
+      - create
+  - apiGroups:
+      - rbac.authorization.k8s.io
+    resources:
+      - clusterroles
+    verbs:
+      - create
+      - list
+      - watch
+  - apiGroups:
+      - authorization.openshift.io
+      - rbac.authorization.k8s.io
+    resources:
+      - roles
+    verbs:
+      - get
+      - create
+  - apiGroups:
+      - authorization.openshift.io
+      - rbac.authorization.k8s.io
+    resources:
+      - rolebindings
+    verbs:
+      - get
+      - update
```

Comment:
It is really necessary that the operator would be able to *update any rolebinding in any namespace* ?

Reply:
Che server for replace us update: https://github.com/eclipse/che/blob/master/infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesWorkspaceServiceAccount.java#L94